crates/common/src/fuzz/mod.rs

```diff
@@ -0,0 +1,266 @@
+use crate::{
+    dotrain_add_order_lsp::LANG_SERVICES,
+    frontmatter::{try_parse_frontmatter, FrontmatterError},
+};
+use dotrain::{error::ComposeError, RainDocument, Rebind};
+use proptest::prelude::RngCore;
+use proptest::test_runner::{RngAlgorithm, TestRng};
+use rain_interpreter_bindings::IInterpreterStoreV1::FullyQualifiedNamespace;
+use rain_interpreter_eval::{
+    eval::ForkEvalArgs,
+    fork::{ForkCallError, ForkedEvm, NewForkedEvm},
+    trace::RainEvalResult,
+};
+use strict_yaml_rust::StrictYamlLoader;
+use thiserror::Error;
+
+#[derive(Clone, Debug)]
+pub struct RunnerScenario {
+    pub name: String,
+    pub entrypoint: String,
```

Comment:
now that we have quote rebinding, i'm not sure that we want/need to specify entrypoints

see https://github.com/rainlanguage/dotrain/issues/91 for a discussion on why/how

orderbook gui always uses the same entrypoints by convention, so introducing different entrypoints at the scenario level breaks the convention and causes the abstraction to leak